src/keymap.cpp

```diff
@@ -139,7 +139,7 @@ static op_desc opdescs[] = {
 	},
 	{
 		OP_SETTAG, "set-tag", "t",
-		_("Select tag"),
+		_("Set tag"),
```

Comment:
Instead of reverting, you can simply remove the commit and force-push the result:

```console
$ git rebase --interactive 9794dd99a25096d809a650c810d62cc5678c895d^
$ git push --force
```